doc/development/deprecations.rst

```diff
@@ -132,6 +132,27 @@ Pending deprecations
   - Deprecated in v0.32
   - Will be removed in v0.33
 
+* The following decorator syntax for transforms has been deprecated:
+
+  .. code-block:: python
+
+      @transform_fn(*transform_args)
+      @qml.qnode(dev)
+      def circuit():
+          ...
+
+  Please call the transform directly using ``circuit = transform_fn(circuit, *transform_args)``,
+  or use ``functools.partial``:
+
+  .. code-block:: python
+
+      @functools.partial(transform_fn, *transform_args)
```

Comment:
Does this way of using partial make an assumption on which argument the QNode should be? The way I read this it looks like the QNode should be the last argument.

Reply:
First. Might be better to document using keyword arguments instead.  Less error-prone to ordering issues if everything is explicitly named.

Reply:
You're right. I'll change it to `**transform_kwargs` to be more explicit.